src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -276,7 +277,8 @@ export class SearchMenu extends React.Component {
               />
               <button
                 type="submit"
-                disabled={!isUserInputValid}
+                disabled={!isUserInputValid()}
+                id="sitewide-search-submit-button"
```

Comment:
Same here 🙂 https://github.com/department-of-veterans-affairs/vets-website/pull/15508/files#r550542199